Pipfile

```diff
@@ -39,6 +39,7 @@ zipp = "~=3.8.1"
 importlib-metadata = "~=4.12.0"
 ruamel-yaml = "~=0.17.21"
 mistune = "~=3.0.1"
+semver = ">=3,<4"
```

Comment:
Should we fixate this to one particular version rather than between 3 and 4?

Reply:
This is to make sure version 4 of semver is not installed. Copied from their website. See https://python-semver.readthedocs.io/en/latest/install.html